smileCounter.py

```diff
@@ -11,6 +11,17 @@
 # To use a video file as input
 # video = cv2.VideoCapture('filename.mp4')
 
+# Text configuration
+font = cv2.FONT_HERSHEY_SIMPLEX
+textPosition = (0,0)
+fontScale = 1
+fontColor = (0,0,0)
+thickness = 3
+lineType = 2
```

Comment:
You mix the notation (snake_case / camelCase) in one file (for example lineType and then smiles_detected)

### We plan to use snake_case in the future, so please replace all variables with proper names.

Reply:
Changed notation of variables